www/react-base/src/components/BuildSticker/__snapshots__/BuildSticker.test.tsx.snap

```diff
@@ -0,0 +1,155 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Not sure those snapshot achieve anything. I don't remember we had UI regression that this would detect.
Maintaining it  means you have to regen them all the time.

Usually, test fixtures are good for reviewers to make sure the test generate appropriate data, but here reading the html do not show you really how it looks like.

Reply:
This was effectively a rewrite of the unit tests we had on the AngularJS implementation side. There we checked explicitly if X element has member Y and so on. Snapshots seem to be better to cover this kind of output because one doesn't need to write the output comparison code at all. The check is for the generated HTML output, not how it would look like. Consequently, tests like these are only useful for low-level components. This also means that we wouldn't need to regenerate the snapshots all that often. I fully agree that for higher-level components snapshot tests don't make sense.